pre-docs/bolt_installing.md

```diff
@@ -13,7 +13,7 @@ Server 12 and Ubuntu 14.04 and 16.04.
 
 Install Bolt and its dependencies on *nix platforms.
 
-1. To install the dependencies you need to use Bolt, run the command that
+1. To install the dependencies you need to install Bolt, run the command that
```

Comment:
This reads ambiguously to me, might be clearer as 'To install dependencies required to install Bolt'

Reply:
I'm not saying anything about building the gem. This is probably just my preference for 3rd person over 2nd person, which others seem to disagree about. I think you can ignore it.